setup.py

```diff
@@ -8,8 +8,9 @@
     "chiabip158==1.1",  # bip158-style wallet filters
     "chiapos==1.0.9",  # proof of space
     "clvm==0.9.7",
+    "clvm_tools==0.4.3",  # Currying, Program.to, other conveniences
```

Comment:
Thanks for comment

Reply:
`Program.to()` comes from `chia-blockchain`. in `chia/types/blockchain_format/program.py`. It relies on `SExp` from `clvm`. Currying might come from `clvm_tools` though